<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KstChangeNptsDialog</class>
 <widget class="QDialog" name="KstChangeNptsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Data Sample Ranges</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Vectors to change:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Clear">
       <property name="whatsThis">
        <string>Unselect all vectors in the list.</string>
       </property>
       <property name="text">
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SelectAll">
       <property name="whatsThis">
        <string>Select all vectors in the list.</string>
       </property>
       <property name="text">
        <string>S&amp;elect All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListBox" name="CurveList">
     <property name="whatsThis">
      <string>The sample ranges for the vectors selected in this list will be changed.</string>
     </property>
     <property name="selectionMode">
      <enum>QListBox::Extended</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="KstDataRange" name="_kstDataRange" native="true"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="Spacer15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Apply">
       <property name="whatsThis">
        <string>Apply new data sample specifications to selected vectors.</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="whatsThis">
        <string>Apply new data sample specifications to selected vectors.</string>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="whatsThis">
        <string>Close dialog without applying further changes.</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KstDataRange</class>
   <extends>QWidget</extends>
   <header>kstdatarange.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CurveList</tabstop>
  <tabstop>Clear</tabstop>
  <tabstop>SelectAll</tabstop>
  <tabstop>_kstDataRange</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>KstChangeNptsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CurveList</sender>
   <signal>selectionChanged()</signal>
   <receiver>KstChangeNptsDialog</receiver>
   <slot>updateTimeCombo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
